parser-typechecker/src/Unison/UnisonFile/Type.hs

```diff
@@ -15,8 +15,8 @@ import Unison.Type qualified as Type
 import Unison.WatchKind (WatchKind)
 
 data UnisonFile v a = UnisonFileId
-  { dataDeclarationsId :: Map v (TermReferenceId, DataDeclaration v a),
-    effectDeclarationsId :: Map v (TermReferenceId, EffectDeclaration v a),
+  { dataDeclarationsId :: Map v (TypeReferenceId, DataDeclaration v a),
+    effectDeclarationsId :: Map v (TypeReferenceId, EffectDeclaration v a),
```

Comment:
`TermReferenceId` and `TypeReferenceId` are aliases for the same type, but clearly these were meant to be `TypeReferenceId`.